chapters/en/chapter10/4.mdx

```diff
@@ -0,0 +1,40 @@
+# Annotate your dataset
+
+🚧 WIP 🚧 
+
+##TODO: Add screenshots!
+
+Now it is time to start working from the Argilla UI to annotate our dataset. 
+
+## Align your team with annotation guidelines
+
+Before you start annotating your dataset, it is always good practice to write some guidelines, especially if you're working as part of a team. This will help you align on the task and the use of the different labels, and resolve questions or conflicts when they come up.
+
+In Argilla, you can go to your dataset settings page in the UI and modify the guidelines and the descriptions of your questions to help with alignment.
```

Comment:
Maybe you could link to this blog post about guidelines you wrote. I guess most of the people don't know much about that.